### ISSUE TITLE
Documentation / reverse engineering of build and integration artifacts 

### ISSUE THREAD

**Issue by htpvu** [eclipse-pass/main#170]
Analysis and Documentation of:

- Deployment artifacts (results of the build process that will be used for deployment)
- Integration artifacts (services, data sets, connection points)

Definition of done:

- A list in [main/docs](https://github.com/eclipse-pass/main/tree/main/docs) of the complete set of build artifacts used to deploy PASS, with a brief description of each.
- A list in [main/docs](https://github.com/eclipse-pass/main/tree/main/docs) of the complete set of integration points where an external system or data set is required for PASS to function as expected, with a brief description of each. Any optional integration points should be included in this list as well and called out as optional.

**Comment by aforward** [comment]
Working through the project list
https://docs.google.com/spreadsheets/d/1Zz0GQy64Sj0-LJjSGd63hMcqJ8LJVe1WLskUg8Bh5_8/edit#gid=0

As well as the terraform repo (github.com/jhu-sheridan-libraries/pass-terraform) for this work.

**Comment by aforward** [comment]
I am tracking each project looking at build, test, deploy
https://github.com/eclipse-pass/main/blob/174-pipeline-documentation/docs/pipeline.md

I might add a separate PR "labelled" against 170 to close this out.

**Comment by aforward** [comment]
As I work through this work, I will push PRs against the relevant repos with updates to URLs and language
https://github.com/eclipse-pass/pass-ui-public/pull/43

**Comment by aforward** [comment]
Another clean up PR
https://github.com/eclipse-pass/pass-docker/pull/261

**Comment by aforward** [comment]
I think once this is merged https://github.com/eclipse-pass/main/pull/223 we can close this ticket as done.
